torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -1970,6 +1970,14 @@ Tensor elu_double_backward(
     }
 }
 
+Tensor sinc_backward(const Tensor& grad, const Tensor& self) {
+  Tensor pi_self = wrapped_scalar_tensor(Scalar(M_PI)) * self;
+  Tensor vals = grad *
+      ((pi_self * pi_self.cos() - pi_self.sin()) / (pi_self * self)).conj();
+  Tensor out = at::zeros_like(self, self.options());
```

Comment:
I still don't think we want to do that.
You can see above in this file how to pass a scalar Tensor to where to avoid creating such a big Tensor full of 0s: https://github.com/pytorch/pytorch/blob/7c15862cc5ffddf9403c53f9a4b667f6ab331586/torch/csrc/autograd/FunctionsManual.cpp#L232